com.unity.ml-agents/CHANGELOG.md

```diff
@@ -36,6 +36,8 @@ empty string). (#4155)
 - Fixed an error when setting `initialize_from` in the trainer confiiguration YAML to
 `null`. (#4175)
 - Fixed issue with FoodCollector, Soccer, and WallJump when playing with keyboard. (#4147, #4174)
+- Fixed a crash in StatsReporter when using threaded trainers with very frequent summary writes
+(#4201)
```

Comment:
```suggestion
- Fixed a rare crash in StatsReporter when using threaded trainers (#4201)
```
Very frequent summary writes weren't a requirement for the crash, it just increased the probability enough so that we could reproduce it reliably.